evals/scripts/criteria.py

```diff
@@ -1,4 +1,5 @@
 count_query_criteria = "The answer must be considered correct if it conveys the same numerical value as the ideal value, regardless of how it is phrased or formatted."
 format_match_criteria = "The answer should list of elements including items like ideal value"
 subset_criteria = "The expected answer should be considered correct if it is a subset of the reference answer, allowing for some items to be included from the list, regardless of order or completeness, even if they contain emoji or ASCII codes."
-tiktok_analysis_criteria = "The answer is not related to ideal value. The answer should provide a concise artist introduction followed by automatically generated fan segment data, including segment names and counts."
\ No newline at end of file
+tiktok_analysis_criteria = "The answer is not related to ideal value. The answer should provide a concise artist introduction followed by automatically generated fan segment data, including segment names and counts."
```

Comment:
Could we include the specific section headers we expect in the output html to make this eval more `specific`? 🎯